src/gatsby-ssr.js

```diff
@@ -1,19 +1,53 @@
+import React from 'react';
 import { ServerStyleSheets } from '@material-ui/styles';
+import postcss from 'postcss';
+import autoprefixer from 'autoprefixer';
+import CleanCSS from 'clean-css';
 
 // Keep track of sheets for each page
 const globalLeak = new Map();
 
+const prefixer = postcss([autoprefixer]);
+const cleanCSS = new CleanCSS();
+
+const defaultOptions = {
+  disableAutoprefixing: false,
+  disableMinification: false,
+};
+
 export const wrapRootElement = ({ element, pathname }, pluginOptions) => {
   const sheets = new ServerStyleSheets(pluginOptions.stylesProvider);
   globalLeak.set(pathname, sheets);
 
   return sheets.collect(element);
 };
 
-export const onRenderBody = ({ setHeadComponents, pathname }) => {
+export const onRenderBody = (
+  { setHeadComponents, pathname },
+  pluginOptions,
+) => {
   const sheets = globalLeak.get(pathname);
+
   if (sheets) {
-    setHeadComponents([sheets.getStyleElement()]);
+    const { disableAutoprefixing, disableMinification } = {
+      ...defaultOptions,
+      ...pluginOptions,
+    };
+
+    let css = sheets.toString();
+
+    css = disableAutoprefixing ? css : prefixer.process(css).css;
+    css = disableMinification ? css : cleanCSS.minify(css).styles;
+
+    setHeadComponents([
+      <style
+        type="text/css"
```

Comment:
This is discouraged by https://validator.w3.org:

![Capture d’écran 2019-05-11 à 14 11 13](https://user-images.githubusercontent.com/3165635/57569532-a8a44e00-73f6-11e9-9d9a-753687af2b10.png)

Reply:
Thanks for the hint. I removed the type attribute